tests/oc-integration-tests/drone/frontend-global.toml

```diff
@@ -72,12 +72,13 @@ multiple = true
 supports_upload_only = true
 
 [http.services.ocs.capabilities.capabilities.files_sharing.public.password]
-enforced = true
+enforced = false
```

Comment:
IMO this is fine. In reva we can run the CI with public.password enforcement off.
In ocis CI, in the future, we will be able to run with various "system parameter" settings, and can have test scenarios that check what happens when various public.password items are enforced.

Reply:
As I understand we can't use the env adjustments in the reva test. We have to cover the OCIS_SHARING_PUBLIC_WRITEABLE_SHARE_MUST_HAVE_PASSWORD configuration the ocis tests.

Reply:
Yes, we now have ways to reset env variables and restart ocis during tests on ocis. So we can do that sort of thing in ocis CI. We don't have any framework to do that is reva with `toml` files etc.